src/applications/vaos/components/Breadcrumbs.jsx

```diff
@@ -1,19 +1,73 @@
 import React, { useEffect, useState } from 'react';
 import PropTypes from 'prop-types';
-import { useLocation } from 'react-router-dom';
+import { useHistory, useLocation } from 'react-router-dom';
 import { VaBreadcrumbs } from '@department-of-veterans-affairs/component-library/dist/react-bindings';
 import { useSelector } from 'react-redux';
 import manifest from '../manifest.json';
 import { getUrlLabel } from '../new-appointment/newAppointmentFlow';
 import { getCovidUrlLabel } from '../covid-19-vaccine/flow';
 import { getPageFlow } from '../referral-appointments/flow';
 
+/**
+ * Base breadcrumb list for all pages in app - this was just separated out into a constant
+ * @type {Array<{href: string, label: string, isRouterLink?: boolean}>}
+ */
+export const BREADCRUMB_BASE = [
+  {
+    href: '/',
+    label: 'Home',
+  },
+  {
+    href: '/my-health',
+    label: 'My HealtheVet',
+  },
+  {
+    // For Breadcrumb objects that are router links, the href is relative to the manifest.rootUrl
+    href: manifest.rootUrl,
+    label: 'Appointments',
+    isRouterLink: true,
+  },
+];
+
+// Just to simplify the regex usage below
+const manifestRootUrlRegex = new RegExp(`^.*${manifest.rootUrl}/?`);
+
+// Either http or https or starts with a slash
+const absoluteUrlRegex = new RegExp('^https?|^/');
+/**
+ * If the href is a router link that is based off the manifest.rootUrl,
+ * we need to make it relative to the manifest.rootUrl
+ * @param {string} href - The href of the breadcrumb - may be relative or absolute
+ * @returns {string} The relative route
+ */
+export const relativeRouteProcessor = (href, addLeadingSlash = true) => {
```

Comment:
Hmm other than in the test, where is addLeadingSlash used?

Reply:
It's not, except in a test. The reason for it would be if we have a router link that is routed not from the root. There, you'd want to pass false so you don't push to the root of the router so it's easier to write a sub-page. Though really we don't do that in breadcrumbs just in other navigation pieces.

Reply:
We could remove it and give a notice that it doesn't support non-leading slashed history pushes.

Reply:
I'll take it out and specify one should always assume/use root-based navigation for router links